jira-macro/jira-macro-charts/src/main/java/org/xwiki/contrib/jira/charts/internal/bidimensionalgrid/JIRABiDimensionalGridChartMacro.java

```diff
@@ -107,7 +115,8 @@ public List<Block> execute(JIRABiDimensionalGridChartMacroParameter parameters,
         }
 
         TableBlock tableBlock = new TableBlock(tableRows);
-        return List.of(tableBlock);
+        return jiraMacroTransformationManager.transform(List.of(tableBlock), parameters, context,
+            jiraServerResolver.resolve(parameters), JIRABiDimensionalGridChartMacro.MACRO_NAME);
```

Comment:
Sounds like it should be part of AbstractMacro no?

Reply:
Yes, if it could avoid making each time the same call in all implems, and to inject same component.